src/cms-content/learn/metric-explainers.json

```diff
@@ -6,7 +6,7 @@
+      "sectionIntro": "A state or county’s community level takes into account three metrics: new COVID admissions per 100,000 population in the past 7 days, the percent of staffed inpatient beds occupied by COVID patients, and total new COVID cases per 100,000 population in the past 7 days.\n\n* [COVID admissions](/covid-risk-levels-metrics#COVID-admissions) and [patients with COVID](/covid-risk-levels-metrics#patients-with-covid) represent the current potential for health system strain.\n* [New COVID cases](/covid-risk-levels-metrics#weekly-new-cases) represents the current amount of COVID in a community and acts as an early warning indicator of potential increases in health system strain.\n\nTo calculate the community level we first assign a level to each of the three metrics based on their values:\n\n* Weekly new cases: <200 is Lower, >=200 is Higher.\n* Weekly hospital admissions per 100k: <10 is Low, 10-19.9 is Medium, >=20 is High\n* Staffed inpatient beds occupied by COVID-19 patients: <10% is Low, 10-14.9% is Medium, >=15% is High.\n\nWe then combine the levels to get the community level as follows:\n\n* If cases are below 200 and both hospital metrics are Low, then the community level is Low.\n* If cases are above 200 and one of the hospital metrics is Medium, then the community level is High.\n* If cases are below 200 then the community level is the higher of the two hospital metrics.\n* If cases are above 200 then the community level is one level worse than the higher of the two hospital metrics. i.e. If both hospital metrics are Low then the community level is Medium. If either hospital metric is Medium or High, then the community level is High.\n\n[Read more details on the CDC's methodology.](https://www.cdc.gov/coronavirus/2019-ncov/science/community-levels.html)\n\nCovid Act Now’s former COVID risk framework was developed in March 2020 in response to the lack of a standardized risk framework across the U.S. [Read about our former COVID risk framework.](/covid-risk-levels-metrics#former-risk-framework)"
```

Comment:
small change that removes a new line that was causing a random backslash to render